tests/docs/test_docs.py

```diff
@@ -29,15 +29,15 @@ def test_html(self):
     #         "%s"%(latex_path)])
     #     assert check == 0
 
-    def test_linkcheck(self):
-        doctrees_path = os.path.sep.join(self.path_to_docs.split(os.path.sep) + ['_build']+['doctrees'])
-        link_path = os.path.sep.join(self.path_to_docs.split(os.path.sep) + ['_build'])
+    # def test_linkcheck(self):
+    #     doctrees_path = os.path.sep.join(self.path_to_docs.split(os.path.sep) + ['_build']+['doctrees'])
+    #     link_path = os.path.sep.join(self.path_to_docs.split(os.path.sep) + ['_build'])
 
-        check = subprocess.call(["sphinx-build", "-nW", "-b", "linkcheck", "-d",
-            "%s"%(doctrees_path),
-            "%s"%(self.path_to_docs),
-            "%s"%(link_path)])
-        assert check == 0
+    #     check = subprocess.call(["sphinx-build", "-nW", "-b", "linkcheck", "-d",
```

Comment:
we are no longer checking links... currently takes to long. we can figure out a strategy for this in a separate pr